spec/support/utilities/iis_application.rb

```diff
@@ -0,0 +1,20 @@
+def has_app(app_name)
+  !(on(default, "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -noninteractive -noprofile -executionpolicy bypass -command \"& {Get-WebApplication -Name #{app_name}}\"").stdout =~ /Started/i).nil?
```

Comment:
These should use the [helper](https://github.com/puppetlabs/puppetlabs-iis/blob/master/spec/support/utilities/utility_helper.rb) so it can work on windows2008